string[] joint_name
float64[] tau_d
float64[] tau_task
float64[] tau_null 